core/src/main/scala/org/apache/spark/executor/TaskMetrics.scala

```diff
@@ -358,5 +374,12 @@ class ShuffleWriteMetrics extends Serializable {
   private[spark] def incShuffleWriteTime(value: Long) = _shuffleWriteTime += value
   private[spark] def decShuffleWriteTime(value: Long) = _shuffleWriteTime -= value
   
-
+  /**
+   * Total number of records written to the shuffle by this task
+   */
+  @volatile private var _recordsWritten: Long = _
```

Comment:
to be more consistent with the presentation in the UI and the other fields here, should this be `shuffleRecordsWritten`?